content/guides/composite_monitors.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Guide to Composite Monitors
+kind: guide
+listorder: 9
+---
+
+***If you're unfamiliar with the basics of Datadog Monitors, first read the [Guide to Monitors](/guides/monitors)***
+
+---
+
+Composite monitors let you combine many individual monitors into one so that you can define more specific alert conditions. Choose two or more existing monitors - monitor A and monitor B, say - and then set a trigger condition using boolean operators (e.g. “A && B”). The composite monitor will trigger when its individual monitors' statuses simultaneously have values that cause the composite's trigger condition to be true.
+
+After you create a composite monitor, you can modify it just as you would an individual monitor. To change the trigger condidtion, edit the composite monitor. To change one of its constituent monitors, though - e.g. to disable its notifications in favor of the composite notification - simply edit that individual monitor.
+
+## Creating composite monitors
+
+In Datadog, go to the New Monitor page and select 'Composite Monitor' from the list of monitor types:
+
+![choose-composite-type](/static/images/composite_monitors/select-monitor-type.png)
+
+### Choose individual monitors
+
+You can choose up to 10 individual monitors to use in the new composite monitor. The individual monitors cannot themselves be composite monitors. 
+
+After you choose your first monitor, the UI will indicate its alert type and current status:
+
+![create-composite-2](/static/images/composite_monitors/create-composite-2.png)
+
+Recall that each Datadog monitor has one of two alert types: simple, or multi-alert. You may create composite monitors from individual monitors of different alert types (e.g. one simple alert monitor and one multi-alert monitor), but if you choose more than one multi-alert monitor, then *all* multi-alert monitors in the composite monitor must use the same grouping. In the screenshot below, monitor 'a' is grouped by `availability-zone`, but monitor 'b' is grouped by `host`:
+
+![create-composite-4](/static/images/composite_monitors/create-composite-4.png)
+
+You cannot create such a composite monitor.
+
+Furthermore, even if all chosen multi-alert monitors use the same grouping, they must *also* have at least one reporting source (i.e. host) in common. In the screenshot below, both monitors are grouped by `host`, yet the UI indicates that the two monitors are incompatible:
```

Comment:
>  they never have the same value

i.e. they don't have any values in common *right now* (hence the warning), but they may in the future, yes?